src/10_functions.py

```diff
@@ -2,8 +2,7 @@
 
 # YOUR CODE HERE
 def is_even(n):
-    if int(n)/2 == int(int(n)/2):
-        return True
+    return int(n)/2 == int(int(n)/2)
```

Comment:
Great work. We can also simply return ` return n % 2 == 0` here. Both work however!